grisera_api/graph_api_service.py

```diff
@@ -27,15 +28,30 @@ def post(self, url_part, request_body):
                                  json=request_body).json()
         return response
 
-    def create_participant(self, participant):
+    def create_participant_node(self):
         """
-        Send to the Graph API request to create participant
-
-        Args:
-            participant (): participant to be created
+        Send to the Graph API request to create participant node
 
         Returns:
             Result of request
         """
         request_body = {"labels": ["Participant"]}
         return self.post("/nodes", request_body)
+
+    def create_participant_properties(self, participant_id: int, participant: ParticipantIn):
+        """
+        Send to the Graph API request to create properties for participant
+
+        Args:
+            participant_id (int): Id of participant
+            participant (ParticipantIn): Participant to be created
+
+        Returns:
+            Result of request
+        """
+        request_body = []
+        for property in participant.dict().items():
+            if property[1] is not None:
+                request_body.append({"key": property[0], "value": property[1]})
+
+        return self.post("/nodes/{}/properties".format(participant_id), request_body)
```

Comment:
What if I would change create_participant_node() and create_participant_properties() to create_node() and create_properties()? Then create_node would take label for node as argument, also create_properties instead of ParticipantIn object would have any object which inherits after BaseModel class.

Reply:
Sounds like a good idea. This way, in `graph_api_service.py `, we'd have generalized _creates_ :+1: